### DISCUSSION TITLE
disableAnimation

### DISCUSSION THREAD

**Discussion by sabrainas** [heroui-inc/heroui#2424] [Help]
disableAnimation does not work in the Popover component.

**Comment by alphaxek** [comment]
Hi @sabrainas, can you please try updating to 2.2.10 version.

I can see its working fine in 2.2.10 version.

1. Without animation:
![popover1](https://github.com/nextui-org/nextui/assets/116849110/cb3cf30a-b1cb-4de5-985a-7aeead9443d2)

2. With animation:
![popover2](https://github.com/nextui-org/nextui/assets/116849110/83820487-9e2d-4a39-8dab-4d5c043d6c7a)


**If this helps, please mark this as answered, thanks!**

**Comment by alphaxek** [comment]
Hi @sabrainas, did it resolve your issue

**Comment by sabrainas** [comment]
Hi @alphaxek! I used `disableRipple` property and it worked. Thank you for your response!

**Reply by alphaxek** [reply]
great thanks!
